documentation/src/main/asciidoc/topics/proc_setting_up_spring_cache.adoc

```diff
@@ -6,14 +6,7 @@ If you use remote caches in a {brandname} Server deployment, you should also con
 
 [IMPORTANT]
 ====
-{brandname} supports Spring version 5 and version 6.
-Be aware that Spring 6 requires Java 17.
-
-The examples in this document include artifacts for the latest version of Spring.
-If you want to use Spring 5 use:
-
-* Remote caches: `infinispan-spring5-remote`
-* Embedded caches: `infinispan-spring5-embedded`
+{brandname} only supports Spring version version 6.
```

Comment:
Don't remove the be aware of kava 17 plz

Reply:
I removed this as we're going to baseline on Java 17 for everything, so I thought I might as well update this while making changes so that it doesn't get missed.